tracer/operation/createcontract.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2024 Fantom Foundation
+// This file is part of Aida Testing Infrastructure for Sonic
+//
+// Aida is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Aida is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with Aida. If not, see <http://www.gnu.org/licenses/>.
+
+package operation
+
+import (
+	"encoding/binary"
+	"fmt"
+	"io"
+	"time"
+
+	"github.com/Fantom-foundation/Aida/state"
+	"github.com/ethereum/go-ethereum/common"
+
+	"github.com/Fantom-foundation/Aida/tracer/context"
+)
+
+// CreateContract data structure
+type CreateContract struct {
+	Contract common.Address
+}
+
+// GetId returns the create-account operation identifier.
+func (op *CreateContract) GetId() byte {
+	return CreateContractID
+}
+
+// NewCreateContract creates a new create-account operation.
+func NewCreateContract(contract common.Address) *CreateContract {
+	return &CreateContract{Contract: contract}
+}
+
+// ReadCreateContract reads a create-account operation from a file.
+func ReadCreateContract(f io.Reader) (Operation, error) {
+	data := new(CreateContract)
+	err := binary.Read(f, binary.LittleEndian, data)
+	return data, err
+}
+
+// Write the create-account operation to file.
+func (op *CreateContract) Write(f io.Writer) error {
+	err := binary.Write(f, binary.LittleEndian, *op)
+	return err
+}
+
+// Execute the create-account operation.
+func (op *CreateContract) Execute(db state.StateDB, ctx *context.Replay) time.Duration {
+	contract := ctx.DecodeContract(op.Contract)
+	start := time.Now()
+	db.CreateContract(contract)
+	return time.Since(start)
+}
+
+// Debug prints a debug message for the create-account operation.
+func (op *CreateContract) Debug(ctx *context.Context) {
+	fmt.Print(op.Contract)
+}
```

Comment:
Documentation seems to be incorrect here (`create-account` -> `create-contract`)?